packages/website/.gitignore

```diff
@@ -18,4 +18,4 @@ yarn-error.log*
 .DS_Store
 
 # ignore generated kit pages
-src/pages/kits/*.md
+src/pages/kits/*.mdx
```

Comment:
Is there anything else that needs _not_ to be ignored in that folder? 🤔 

If previously these files were md files, and somebody in their local branches pull down the new changes, the lingering md files from the previous version can be accidentally committed. Therefore, I'd suggest adding the .mdx rule and keeping the .md as well, or just ignore the whole folder content if everything there should be ignored.

What do you think?